### DISCUSSION TITLE
Can't Directly Upload Documentation

### DISCUSSION THREAD

**Discussion by colcrunch** [community/community#112510] [GitHub Education]
### Hi everyone,

Why is there absolutely no way to directly upload a pdf or image of a document? What is the point of accepting enrollment documents which almost no one has in paper form if there is no efficient way to submit them?

The current system assumes that you are using a computer with a working webcam, and doesn't even offer the ability to swap to your phone to complete the process with a working camera like other ID verification platforms like ID.me does. I would love to reverify my eligibility, but this system is asinine.

**Comment by narrow-mind** [comment]
I cropped PDF and converted to image file and it worked.

**Comment by samus-aran** [comment]
Hi @colcrunch  👋

We're happy you're here!

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. You are able to submit your application through a mobile device. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance

**Comment by f-semeria** [comment]
I think I found a bug in the proof upload page.
PNGs seem not to be actually accepted (once clicking "process application" I'm sent again to the same proof upload page with no banner or other warning.

Document was under 1Mb and above the minimum resolution. I tried both Chrome and Mozilla browsers.
I solved it by simply saving it as a JPG, but I think it should be fixed.
I hope this is the right place to send this.

Best
